sys/net/application_layer/dhcpv6/client.c

```diff
@@ -144,10 +144,30 @@ void dhcpv6_client_auto_init(void)
 static void *_thread(void *args)
 {
     (void)args;
+    pid_t netif_pid = SOCK_ADDR_ANY_NETIF;
```

Comment:
How does that work?

Reply:
Same as before. I haven't changed this behavior.